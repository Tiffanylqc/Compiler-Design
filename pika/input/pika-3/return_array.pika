func f<>->[float] {
	return [1.3, 7.6, 8.2].
}

func g<>->[float] {
	return [1.4, 7.0, 8.9].
}


exec {
	print f(), _n_.
	print g().
}