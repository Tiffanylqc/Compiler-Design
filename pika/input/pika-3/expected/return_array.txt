[1.3, 7.6, 8.2]
[1.4, 7, 8.9]